app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -2665,6 +2665,7 @@ class BrowserTabViewModel @Inject constructor(
     suspend fun refreshCta(): Cta? {
         if (currentGlobalLayoutState() is Browser) {
             val isBrowserShowing = currentBrowserViewState().browserShowing
+            val isErrorShowing = currentBrowserViewState().maliciousSiteDetected
```

Comment:
I couldn't reproduce this issue on SSL error view, since omnibar enters edit mode when SSL error view is displayed. I changed that behavior for malicious sites error view, but not sure if we want to change it for SSL as well. Pinging @malmstein just in case

Reply:
We can leave it as is for now, and think about the general approach when we consolidate these pages.